packages/cashier/src/Containers/__tests__/p2p-cashier.spec.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import P2PCashier from '../p2p-cashier';
+import { createBrowserHistory } from 'history';
+import { Router } from 'react-router';
+import { routes } from '@deriv/shared';
+
+jest.mock('Stores/connect.js', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    connect: () => Component => Component,
+}));
+
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    Loading: () => <div>Loading</div>,
+}));
+
+jest.mock('@deriv/p2p', () => () => <div>P2P</div>);
+
+describe('<P2PCashier />', () => {
+    it('should render <Loading /> component', () => {
+        const history = createBrowserHistory();
```

Comment:
@george-usynin-deriv Could you define `history` at the top one time and use it in the tests

Reply:
sure @Bahar-FS, thx for the notice